metatron-app/metatron-discovery
# #3720 Add numeric format for each measure in kpi chart

### Description
Added number format setting for each measure in KPI

**Related Issue** : #3720 


### How Has This Been Tested?
1. Create a KPI chart in the Create Chart screen.
2. Change the number format in the Dimension Value field on the shelf.
3. Check if it is displayed according to the changed format.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
